Utilities/Scripts/runCodespell.sh

```diff
@@ -31,7 +31,8 @@ codespell -q6 --skip="\
 ./Libs/vtkITK/itkLevelTracingImageFilter.h,\
 ./Modules/CLI/ExtractSkeleton/tilg_iso_3D.cxx,\
 ./Modules/CLI/ResampleDTIVolume/itkDiffusionTensor3D*,\
-./Modules/Loadable/Reformat/Resources/UI/qSlicerReformatModuleWidget.ui\
+./Modules/Loadable/Reformat/Resources/UI/qSlicerReformatModuleWidget.ui,\
+./Utilities/Scripts/runCodespell.sh\
```

Comment:
Could review and tweak the commit message based on the following suggestion:

```
ENH: Exclude runCodespell script from checking itself 

This is needed to avoid false positive due to list of exceptions listed in the script
```

Reply:
I updated the commit message to your suggestion.